assig1/assignment-1/src/main/java/ro/utcn/spet/example/a1/entity/Tag_question.java

```diff
@@ -0,0 +1,29 @@
+package ro.utcn.spet.example.a1.entity;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Tag_question {
```

Comment:
Java naming conventions: use camelCase instead of low_snake (i.e. TagQuestion instead of Tag_question).